test/kitchen/platforms.json

```diff
@@ -33,8 +33,8 @@
             "x86_64": {
                 "debian-8": "urn,credativ:Debian:8:8.0.201901221",
                 "debian-9": "urn,credativ:Debian:9:9.20200722.0",
-                "debian-10": "urn,Debian:debian-10:10:0.20230222.1299",
```

Comment:
We are not the only ones to use this, maybe we should duplicate it to use a CWS specific version ? I think the other is fine since it's not a daily version

Reply:
Good point, I'll do that